debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/valueholder/LogMinerDmlEntry.java

```diff
@@ -89,4 +94,8 @@
      */
     void setTransactionId(String id);
 
+    /**
+     * @param rowId unique row identifier
+     */
+    void setRowId(String rowId);
```

Comment:
Is it realistic to make this class immutable? If so, could you log a separate issue for this one?

Reply:
It's probably realistic with some refactoring.  We're using the `LogMinerDmlEntry` that is returned by the parser to store these values but perhaps instead we should simply have another wrapper object around LogMinerDmlEntry that is immutable and takes this parser return object as a ctor arg as well.  I'll add a follow-up jira for this.

Reply:
Ok, let's discuss in that issue then. We may have to weigh it against object creation overhead, so perhaps the mutable approach _is_ better after all in this case.

Reply:
I'm also using `LogMinerDmlEntry` with the new `SelectLobLocatorParser` that is part of the CLOB work as well.  Most of the information like setting row-id, etc can be made immutable but the collections which hold the column name/values cannot.  The CLOB/BLOB work requires that we take each event and potentially merge it with a prior event in the transaction stream for correct event emission.  Hopefully when you're back we can circle back and look at that work in conjunction with this to see whether we can decide on a different approach or if what we're doing is fine.